test/test_associations.rb

```diff
@@ -228,7 +228,7 @@ def test_double_belongs_to_with_deleted
   end
 
   def test_mass_assignment_of_paranoid_column_enabled
-    if ActiveRecord::VERSION::MAJOR > 4 && ActiveRecord::VERSION::MINOR > 1
```

Comment:
Ahh ha, this is how you do a gem version comparison!  I knew there was a better way than what I did.